src/pieces/commands/onboarding.py

```diff
@@ -0,0 +1,191 @@
+    steps = {
+        "Step 1: Creating a snippet":[
+            OnboardingStep("Let's get started by saving a snippet\n"
+                    "copy the following snippet python snippet: \n"
+                    f"```python\n{demo_snippet}\n```",
+                    create_snippet_one_validation
+                ),
+            OnboardingCommandStep(
+                "You can save a snippet by typing `pieces create`",
+                "pieces create"
+            )
+        ],
+        "Step 2: Opening the created snippet":[
+            OnboardingCommandStep(
+                "Now let's checkout all the saved snippets by typing `pieces list`",
+                "pieces list"
+            )
+        ],
+        "Step 3: Run in loop":[
+            OnboardingStep(
+                "Running in loop help you to create a session with the Copilot which we will be convering " 
+                "or perform an opration on the same snippet multiple times which will increase your **productivity**",
+                lambda: (True, "")
+            ),
+            OnboardingCommandStep(
+                "You can run in loop by typing `pieces run` Don't forget to exit the loop by typing `exit`",
+                "pieces run"
+            )
+        ],
+        "Step 5: Chat with the Copilot":[
+            OnboardingCommandStep(
+                "Now you understood what is `pieces run` let's try to interact with the Copilot by typing `pieces ask 'How to print I love Pieces CLI in Python and Java'`",
+                "pieces ask 'How to print I love Pieces CLI in Python and Java'"
+            ),
+            OnboardingCommandStep(
+                "Well you just asked one question in a conversation, " "Let's create a session with Copilot by typing `pieces run` then use `ask` to interact with Copilot",
+                "pieces run"
+            ),
+            
+        ],
+        "Step 6: Sharing your feedback" : [
+            OnboardingCommandStep(
+                "Your feedback is very **important** for us to improve feel free to share your feedback by typing `pieces feedback`",
+                "pieces feedback"
+            )
+        ],
+        "Step 7: Contributing":[
+            OnboardingCommandStep(
+                "Pieces is a **open source project** and you can contribute to it by creating a pull request or open an issue by typing `pieces contribute`",
+                "pieces contribute"
+            )
+        ]
+    }
+    console.print("Welcome to Pieces CLI")
+    console.print("Whenever you want to exit the onboarding flow type `exit`")
+    console.print("Remeber anything and Everything")
+    if not Settings.pieces_client.open_pieces_os():
+        console.print("❌ Pieces OS is not running")
+        console.print(
+            Markdown(
+                "# Pieces OS\n\n"
+                "**Pieces OS** is a background service" 
+                " powering Pieces CLI and operating all Pieces services such as IDE integrations:\n\n"
```

Comment:
"**Pieces OS** is a background service" 
                " that powers Pieces CLI and all the other Pieces Integrations such as IDE plugins:\n\n